ethstore/src/account/crypto.rs

```diff
@@ -79,12 +79,14 @@ impl Crypto {
 
 	/// Encrypt custom plain data
 	pub fn with_plain(plain: &[u8], password: &Password, iterations: u32) -> Result<Self, crypto::Error> {
-		let salt: [u8; 32] = Random::random();
+		let salt_arr: [u8; 32] = Random::random();
```

Comment:
Yeah, I wonder the same since [crypto::derive_key_iterations](https://github.com/paritytech/parity-common/blob/master/parity-crypto/src/lib.rs#L58) requires a slice and it should be fine to allocate an array on the stack and pass a reference to it!

Reply:
bc `Pbkdf2` has `salt` as `Vec` now. But ok, I can pass `[u8]`  into [`crypto::derive_key_iterations`](https://github.com/paritytech/parity-common/blob/master/parity-crypto/src/lib.rs#L58)